src/openrct2/ride/Ride.cpp

```diff
@@ -1814,7 +1817,6 @@ static void ride_music_update(Ride* ride)
             ride->music_tune_id = static_cast<uint8_t>(util_rand() % numTracks);
             ride->music_position = 0;
         }
-        return;
```

Comment:
Why did you remove this return?

Reply:
Oh, that's strange. I didn't see this in Github Desktop when I reviewed.

I'll revert.